drivers/include/periph/qspi.h

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (c) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_periph_qspi Quad SPI peripheral
+ * @ingroup     drivers_periph
+ * @brief       Low-level QSPI peripheral driver
+ */
+
+#ifndef PERIPH_QSPI_H
+#define PERIPH_QSPI_H
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <limits.h>
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Serial Flash JEDEC commands
+ */
+typedef enum {
```

Comment:
I don't think this belongs here. In the NOR case this is not in the cpu's spi case, and we could use this module with chips that don't necessarily conform to these registers.